.github/workflows/pr-minikube.yaml

```diff
@@ -1,6 +1,7 @@
 name: CI-Minikube
 on:
-  - pull_request
```

Comment:
Let's not remove this trigger just yet. We can keep both the GH actions for now.